dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/WorkflowExecuteRunnable.java

```diff
@@ -2078,21 +2063,27 @@ private boolean isNewProcessInstance() {
     }
 
     public void resubmit(long taskCode) throws Exception {
-        ITaskProcessor taskProcessor = activeTaskProcessorMaps.get(taskCode);
-        if (taskProcessor != null) {
-            taskProcessor.action(TaskAction.RESUBMIT);
-            log.debug("RESUBMIT: task code:{}", taskCode);
-        } else {
-            throw new Exception("resubmit error, taskProcessor is null, task code: " + taskCode);
-        }
+        // todo: resubmit
+        // taskExecuteRunnableMap.get(taskCode).resubmit();
+        // ITaskProcessor taskProcessor = activeTaskProcessorMaps.get(taskCode);
+        // if (taskProcessor != null) {
+        // taskProcessor.action(TaskAction.RESUBMIT);
+        // log.debug("RESUBMIT: task code:{}", taskCode);
+        // } else {
+        // throw new Exception("resubmit error, taskProcessor is null, task code: " + taskCode);
+        // }
     }
 
     public Map<Long, Integer> getCompleteTaskMap() {
         return completeTaskMap;
     }
 
-    public Map<Long, ITaskProcessor> getActiveTaskProcessMap() {
-        return activeTaskProcessorMaps;
+    // public Map<Long, ITaskProcessor> getActiveTaskProcessMap() {
```

Comment:
It seems that many commented places have not been deleted, and need to be checked completely.

Reply:
I have removed `ITaskProcessor`